gremlin/traversal/cflows.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package traversal
+
+// #cgo CFLAGS: -I../../probe/ebpf
+// #include "flow.h"
+import "C"
+import (
+	"math/rand"
+	"net"
+	"time"
+	"unsafe"
+
+	"github.com/skydive-project/skydive/flow"
+)
+
+func newICMPFlow(id uint32, nodeTID string, linkSrc string, linkDst string) *flow.EBPFFlow {
```

Comment:
can you move this in the test file as it is only used for the tests

Reply:
The reason this funtion is in a seperate file is that cgo is not allowed in test files. i.e. files ending with _test.go
so this is my work around for this. If you have a better idea would love to hear it;)

Reply:
That is indeed a good reason :)